plugin/al/lib/AL_USDMaya/AL/usdmaya/fileio/TransformIterator.cpp

```diff
@@ -179,7 +184,7 @@ UsdPrim TransformIterator::parentPrim() const
     UsdPrim parentPrim = m_stage->GetPseudoRoot();
     if (m_primStack.size() > 1) {
         parentPrim = (m_primStack.end() - 2)->m_prim;
-        if (parentPrim.IsMaster()) {
+        if (parentPrim.IsPrototype()) {
```

Comment:
There is no guard against PXR_VERSION < 2011?

Reply:
Nice catch, thank you, this was my fault for not testing thoroughly enough before pushing. I'll make sure to start building and testing against both USD's release version and latest version prior to opening a pull request.